File: atcoder.jp/contests/abc287/tasks/abc287_a/main.hs
import Control.Monad (replicateM)
import Util (count)

main :: IO ()
main = do
  n <- readLn
  s <- replicateM n getLine

  let ans = n `div` 2 < count (== "For") s

  putStrLn $ if ans then "Yes" else "No"

File: atcoder.jp/contests/abc287/tasks/abc287_a/main.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
  int N;
  cin >> N;
  vector<string> S(N);
  for (auto &&s : S) cin >> s;

  bool ans = N / 2 < count(S.begin(), S.end(), "For");

  cout << (ans ? "Yes" : "No") << endl;
}
